src/app/Library/CrudPanel/Traits/ColumnsProtectedMethods.php

```diff
@@ -174,6 +184,24 @@ protected function makeSureColumnHasEntity($column)
         return $column;
     }
 
+    /**
+     * Relation methods does not require parameters.
+     *
+     * @param object $model
+     * @param string $method
+     * @return bool
+     */
+    private function makeSureMethodDoesNotRequireParameters($model, $method)
```

Comment:
@pxpm I have some suggestions over this naming 😅

```suggestion
    private function doesMethodHasParameters($model, $method)
```

Since this returns a `bool` I would name it with a starting verb, thus, `does`.
Also, I wouldn't add the `not`, because it may lead to situations (as we actually do, above) where we are double negating;

`if(isNotAbc())`, should be `if(! isAbc())`.
`if(! isNotAbc())`, should be `if(isAbc())`.

`HasParameters` may be `TakesParameters`, or anything else 🤷‍♂️